.goreleaser.yaml

```diff
@@ -0,0 +1,494 @@
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+    - make bootstrap
+
+builds:
+  - id: builds-linux
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{ .CommitDate }}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - riscv64
+    goarm:
+      - "6"
+      - "7"
+    ignore:
+      - goos: darwin
+        goarm: "6"
+      - goos: freebsd
+        goarm: "7"
+      - goos: illumos
+        goarch: arm
+      - goos: illumos
+        goarch: arm64
+      - goos: linux
+        goarm: "7"
+      - goos: openbsd
+        goarch: riscv64
+      - goos: openbsd
+        goarm: "7"
+      - goos: solaris
+        goarch: arm
+      - goos: solaris
+        goarch: arm64
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    skip: false
+  - id: builds-other
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{ .CommitDate }}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+      #- illumos
+      - windows
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - riscv64
+    goarm:
+      - "6"
+      - "7"
+    ignore:
+      - goos: darwin
+        goarm: "6"
+      - goos: darwin
+        goarch: arm
+      - goos: darwin
+        goarch: riscv64
+      - goos: freebsd
+        goarm: "7"
+      - goos: illumos
+        goarch: arm
+      - goos: illumos
+        goarch: arm64
+      - goos: linux
+        goarm: "7"
+      - goos: openbsd
+        goarch: riscv64
+      - goos: openbsd
+        goarm: "7"
+      - goos: solaris
+        goarch: arm
+      - goos: solaris
+        goarch: arm64
+      - goos: windows
+        goarch: riscv64
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    skip: false
+
+report_sizes: true
+
+nfpms:
+  - vendor: OpenBao
+    homepage: https://github.com/openbao/openbao
+    maintainer: OpenBao
+    description: |
+      OpenBao exists to provide a software solution to manage, store, and distribute 
+      sensitive data including secrets, certificates, and keys.
+    license: MPL-2.0
+    formats:
+      # - apk
+      - deb
+      - rpm
+      #- archlinux
+    dependencies:
+      - openssl
+    provides:
+      - bao
+    bindir: /usr/bin
+    section: default
+    contents:
+      - src: ./LICENSE
+        dst: /usr/share/doc/openbao/copyright
+        file_info:
+          mode: 0644
+      - src: ./.release/linux/package/etc/openbao/openbao.env
+        dst: /etc/openbao/openbao.env
+        file_info:
+          mode: 0644
+      - src: ./.release/linux/package/etc/openbao/openbao.hcl
+        dst: /etc/openbao/openbao.hcl
+        file_info:
+          mode: 0644
+      - src: ./.release/linux/package/usr/lib/systemd/system/openbao.service
+        dst: /usr/systemd/system/openbao.service
```

Comment:
Based on `openbao-2.0.0~alpha20240329-1.x86_64.rpm` from https://github.com/openbao/openbao/actions/runs/8482716317, this should be at `/usr/lib/systemd/system/openbao.service`; the **`/lib`** is missing.

Reply:
Good catch, thanks. I will get this fixed.